zookeeper-server/src/main/java/org/apache/zookeeper/server/FinalRequestProcessor.java

```diff
@@ -150,6 +139,24 @@ public void processRequest(Request request) {
             }
         }
 
+        return rc;
+    }
+
+    public void processRequest(Request request) {
+        LOG.debug("Processing request:: " + request);
```

Comment:
Please use parameterized logging format here - I believe that in its current form the string concatenation will happen as the parameter has to be evaluated first, even when debug level logging is disabled. Since this is on critical path of request processing, the performance will take a hit.

Reply:
Update -  I realized that this was not new code, so my previous comment is non blocking for landing the PR. But it's still good to update the log format since we are here...

Reply:
done.